src/command/FileCommands.cxx

```diff
@@ -234,9 +236,93 @@ handle_read_comments(Client &client, Request args, Response &r)
 	gcc_unreachable();
 }
 
+AllocatedPath
+find_art(AllocatedPath directory)
+{
+    std::vector<std::string> art_names = {
```

Comment:
An array would be better:

    static constexpr const char art_names[] = {...

.. because it can be filled at compile time, and requires no run-time code at all. Additionally, your version allocates memory at runtime for both `std::vector` and each `std::string` instance (in addition to the compile-time memory allocated for the same data).

No big deal, that's a simple thing to change - maybe we should postpone small things like that until the big things are done.